_data/simple-icons.json

```
@@ -1181,6 +1181,12 @@
             "source": "https://www.avast.com/",
             "guidelines": "https://press.avast.com/media-materials"
         },
+        {
+            "title": "Avira",
+            "hex": "DB0D15",
+            "source": "https://www.avira.com/en",
```

I think the source URL should be https://www.avira.com/en/media-library#our-logos as well because that's where the ZIP archive containing the logos can be downloaded.

```suggestion
            "source": "https://www.avira.com/en/media-library#our-logos",
```